### ISSUE TITLE
Training Tiny YOLO v2, need some clarifications 

### ISSUE THREAD

**Issue by sfarkya** [leeyoshinari/YOLO_v2#27]
Hello @leeyoshinari 

I am trying to train Tiny Yolo v2 model on VOC dataset ([https://github.com/pjreddie/darknet/blob/master/cfg/yolov2-tiny.cfg](https://github.com/pjreddie/darknet/blob/master/cfg/yolov2-tiny.cfg)) with your training procedure.
 
First, I tried to train from scratch, I got NaNs after few iterations while using momentum optimizer (I also tried different learning rate) but while using adam optimizer my model is still training without NaNs. However, I am not getting correct predictions. 

So, now, I am trying to start training using Imagenet weights for the Tiny Yolo v2, I got the pretrained weights from here [https://pjreddie.com/darknet/imagenet/#reference](https://pjreddie.com/darknet/imagenet/#reference) . Can you please suggest me what would be the ideal training setting according to you for learning rate and optimizer to train properly?

Also, how did you get pre-trained Imagenet weights for the models you have uploaded here? Did you extract it from darknet or you trained them by yourself?  

Thank you. 
Best, 
Saurabh
